components/BenefitsList/BenefitsList.tsx

```diff
@@ -0,0 +1,27 @@
+import type { NextPage } from "next";
+import Benefit from "components/Benefit/Benefit";
+import styles from "styles/BenefitsList.module.scss";
+
+interface Benefit {
+    text: string;
+}
+
+type BenefitsListProps = {
+    heading: string;
+    benefits: Benefit[];
+};
+
+const BenefitsList: NextPage<BenefitsListProps> = ({ heading, benefits }) => {
+    return (
+        <div className={styles.container}>
+            <h3 className={styles.heading}>{heading}</h3>
```

Comment:
Note based on the example usage in the homepage, this means we’ll have an invalid heading hierarchy (going straight from h1 to h3). We might want to make the heading level configurable? Or switch this to `h2`?